### PR TITLE
Serenity

### PR DESCRIPTION
**Pull request by trufae**

### CODE REVIEW

File: vlib/v/pref/os.v [hunk 1]
```diff
@@ -88,6 +91,9 @@ pub fn get_host_os() OS {
 	$if dragonfly {
 		return .dragonfly
 	}
+	$if serenity {
```
Comment (spytheman) [inline vlib/v/pref/os.v:94]: this should be done in a separate PR
the current one just introduces it, but should not use it, so that v.c can be regenerated after its merging

Comment (spytheman) [reply vlib/v/pref/os.v:94]: in the second PR, since the v.c code will have been regenerated, the V produced by it, can understand what `$if serenity {` means.

Comment (trufae) [reply vlib/v/pref/os.v:94]: Its fine if we commit this commented? If not ill update the pr when i come back to the laptop
